docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-COMBINE.xml

```diff
@@ -209,57 +241,63 @@ nameRec := RECORD
 END;
  
 resultRec := RECORD(inrec)
-  DATASET(nameRec) names;
+  DATASET(nameRec) 
```

Comment:
Why the newline here?

Reply:
@gavin I removed the newline. Back to you